rq/job.py

```diff
@@ -322,18 +322,18 @@ def get_position(self) -> Optional[int]:
             return q.get_job_position(self._id)
         return None
 
-    def get_status(self, refresh: bool = True) -> JobStatus:
+    def get_status(self, refresh: bool = True) -> Optional[JobStatus]:
```

Comment:
I think `get_status()` should always returns something. If the job hash in Redis has no status, raise an exception.

Reply:
Thank you for the note. I updated it to raise an exception if there's no status in Redis. I added a `status=JobStatus.QUEUED` to a few `Job.create` calls in other tests so that they wouldn't fail as a result. I don't think this changed the purpose or accuracy of those tests.